MNE/disp3D/3DObjects/brain/brainrtsourcelocdatatreeitem.h

```diff
@@ -211,9 +211,9 @@ class DISP3DNEWSHARED_EXPORT BrainRTSourceLocDataTreeItem : public AbstractTreeI
     /**
     * This function set the normalization value.
     *
-    * @param[in] dNormalization     The new normalization value.
+    * @param[in] vecThresholds     The new threshold values used for normalizing the data.
     */
-    void setNormalization(double dNormalization);
+    void setNormalization(const QVector3D& vecThresholds);
 
 public slots:
```

Comment:
don't use slots anymore (obsolete) just connect with new connect syntax to member functions. so just public: